.github/workflows/linting.yml

```diff
@@ -14,7 +14,9 @@ jobs:
       group: ${{ github.workflow }}-${{ github.ref }}
     steps:
       - uses: actions/checkout@v2
-      - name: Run mdl
-        uses: actionshub/markdownlint@main
+      - uses: ruby/setup-ruby@v1
         with:
-          filesToIgnoreRegex: "SUMMARY.md"
+          ruby-version: '3.1'
+
+      - run: gem install mdl --version 0.11.0
```

Comment:
Why is this gem not specified in the Gemfile?

Reply:
Because there is no Gemfile. I can add one if that would be cleaner.

Reply:
Project doesn't have a Gemfile nor a Ruby version specified.

Reply:
Yes, please add one. Otherwise the CI may lint differently than I do locally.

Reply:
Maybe it would indeed be clever to add a Gemfile and a `bin/check` script so one could also run mdl locally?

Reply:
Yes. Agree to both Gemfile and bin/check

Reply:
Done.